examples/day_in_history.js

```diff
@@ -52,15 +52,20 @@ function ingestArticleProfile(hatch, uri) {
             dayInHistoryYear: dayInHistoryYearFinal
         });
 
+        // TODO: Convert to v2.0 API
         asset.set_document(content);
         asset.set_section("day_in_history");
 
         hatch.save_asset(asset);
+    })
+    .catch(err => {
+        console.error(err.stack);
+        throw err;
     });
 }
 
 function main() {
-    const hatch = new libingester.Hatch();
+    const hatch = new libingester.Hatch('day_in_history', 'en', { argv: process.argv.slice(2) });
```

Comment:
The argv processing we can integrate into the libingester (I originally thought of it being used that way) but the problem is that we don't know how the program was invoked and if cli args are intended for the ingester and not the libingester. On top of this, making every ingester handle parsing of libingester-specific args seemed like a huge hurdle for writers of the code. I think we have to leave that decision on how many process args to trim off to the actual ingester but if we can think of a better way to handle this, I'm all about it.